iOSApp/Screens/Main/HorizontalRailsTableViewCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  HorizontalRailsTableViewCell.swift
+//  iOSApp
+//
+//  Created by Inas Ahmic on 31. 3. 2021..
+//
+
+import UIKit
+
+class HorizontalRailsTableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionViewDataSource {
+
+    
+    @IBOutlet weak var collectionView: UICollectionView!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        collectionView.delegate = self
+        collectionView.dataSource = self
+        collectionView.register(UINib(nibName: "HorizontalRailsCollectionViewCell", bundle: nil), forCellWithReuseIdentifier: "HorizontalRailsCollectionViewCell")
+       
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return 3
+    }
+    
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "HorizontalRailsCollectionViewCell", for: indexPath) as! HorizontalRailsCollectionViewCell
+        
+            return cell
+    }
+    
+    
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        collectionView.layer.cornerRadius = 10
+        collectionView.layer.masksToBounds = true
```

Comment:
this can go in `awakefromnib` function becuase it is related to setup of collection view and not collection view cell.
Also, how the table view delegate method appeared in the middle of collection view delegate methods ? :D